### DISCUSSION TITLE
Sidebar with "@bind"ed variables

### DISCUSSION THREAD

**Discussion by dmetivie** [fonsp/Pluto.jl#2562] [Pluto development] [answered]
When I have a lot of `@bind` variables defined all around the notebook, it is hard to play with them (having to scroll up to change and then scroll down to see the result).
Sure, I could place them all at the same place, but it does not seem appropriate (they are not defined at the same moment, for example).
 
Could it be possible to have (similarly to Table of Content) a side panel with the `@bind` we want (we could decide that with a keyword).
Like that we can simply slide, check, select etc. from all of them next to the one plot we are interested in and directly see the result. 
Thanks.
Hope this is not a repost

**Comment by disberd** [comment, marked as answer]
Have a look at https://github.com/disberd/PlutoExtras.jl#bondtable-new and the example functionality at [the linke static notebook](https://rawcdn.githack.com/disberd/PlutoExtras.jl/94fdc84217b5591343ba4671666511d16b1c0eaf/html_exports/test_bondstable.jl.html).

It is still a draft implementation and I'd have to polish it up when I have a bit more time but it was basically created for that reason

**Comment by aplavin** [comment]
Also see https://github.com/JuliaPluto/PlutoUI.jl/pull/257 for a very simple sidebar implementation. Can have multiple sidebars with different inputs/outputs at different locations, turns out pretty convenient IME!

**Comment by dmetivie** [comment]
Amazing jobs. Both approach looks cool. The `bondtable `look exactly how I imagined it.
Thanks
